openbase/bco
# Feature/#62 openhab 4.0.4 support

### :scroll: Description

Implements latest openhab support.

### Test Procedure

**Be aware that this PR is already build on top of the new mqtt setup described in the `bco.website -> main` repo.**

* setup new bco docker setup (https://github.com/openbase/bco.website/blob/main/www/user/howto/setup-docker.md)
  * use `review` as version for all bco instances
  * upgrade openhab to 4.0.4
    * (it could be that the hue bridge communication needs no be switched from https to http in order to get the hue bridge back on track) 
  * verify device control via bco.
  * verify that you get sensor events.
